context/app/static/js/components/detailPage/files/Files/Files.jsx

```diff
@@ -4,13 +4,12 @@ import PropTypes from 'prop-types';
 import DetailContext from 'js/components/detailPage/context';
 import SectionHeader from 'js/shared-styles/sections/SectionHeader';
 import { DetailPageSection } from 'js/components/detailPage/style';
-import GlobusSection from '../GlobusSection';
 import FileBrowser from '../FileBrowser';
 import FileBrowserDUA from '../FileBrowserDUA';
 import FilesContext from './context';
 import { MarginBottomDiv } from './style';
 
-function Files({ files, uuid, hubmap_id, visLiftedUUID }) {
+function Files({ files }) {
```

Comment:
Are these props no longer needed? We're still passing these props from `Dataset` to this component. 

I believe we would still want to use `visLiftedUUID` to show information for the dataset `Support` entities. We should discuss with @tsliaw.

Reply:
I realized I forgot about the situation in which we are showing the Globus links for support dataset. Please do not remove these props until I make the mockup for how that should appear in the new "Bulk Data Transfer" section.  Other than that, I'm not sure if the props are being used for other purposes such as pulling in the related donor/sample metadata.

Reply:
In the meantime, we should this component should be restored to its original state pending its full replacement by the BulkDataTransfer component.

Reply:
Update - `visLiftedUUID` and the related logic is now handled in `GlobusLinkContainer` (defined in the same file as the `GlobusLink`, so this concern should be resolved 👍